snowshu/adapters/target_adapters/base_target_adapter.py

```diff
@@ -193,7 +192,9 @@ def _get_data_type(self, source_type: str) -> DataType:
     @staticmethod
     def _build_snowshu_envars(snowshu_envars: list) -> list:
         """helper method to populate envars with `snowshu`"""
-        return [f"{envar}=snowshu" for envar in snowshu_envars]
+        envars = [f"{envar}=snowshu" for envar in snowshu_envars]
+        envars.append(f"PGDATA=/{DOCKER_REMOUNT_DIRECTORY}")
+        return envars
```

Comment:
Let's refactor the envar setup to work the way we were discussing earlier, where the keys and values are set by the more concrete type of the adapter.

Only the postgres adapters are going to care about the `PGDATA` envar.